.github/workflows/ci.yaml

```diff
@@ -2,7 +2,7 @@ name: CI
 on:
   push:
     tags:
-      - 'v*.*.*'
+      - "v*.*.*"
```

Comment:
Hope this is intentional

Reply:
That's some auto format of the editor. I did not notice but it does not matter anything